<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newArticleWorkspaceWindow</class>
 <widget class="QDialog" name="newArticleWorkspaceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Article Workspace</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Article Title:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Writer:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="articleTitleTextField">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="placeholderText">
    <string>Add Article Title...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Section:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descriptionTextField">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>501</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="placeholderText">
    <string>Add Article Description...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Photographer:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sectionComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="writerComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="photographerComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Article File:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseFile_pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose File...</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="articleFileTextField">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="placeholderText">
    <string>Choose File to Submit...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submit_pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>580</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Issue Date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="issueDateEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>500</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2015</year>
     <month>2</month>
     <day>28</day>
    </datetime>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>ddd MMM d, yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2015</year>
     <month>2</month>
     <day>28</day>
    </date>
   </property>
  </widget>
  <widget class="QPushButton" name="addImage_pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>380</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Image...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Images:</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="img_scrollArea">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>359</width>
      <height>99</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="delete_pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>450</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteAWS_pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Workspace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyHistory_pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>270</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>View Copy History...</string>
   </property>
  </widget>
  <widget class="QLabel" name="issueDescription">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setCurrentIssueButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>500</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Current Issue</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
